packages/common/src/core/types/IHubSurvey.ts

```diff
@@ -0,0 +1,10 @@
+import { IWithPermissions } from "../traits";
+import { IHubItemEntity, IHubItemEntityEditor } from "./IHubItemEntity";
+
+/**
+ * Defines properties of Hub Survey object
+ * @internal
+ */
+export interface IHubSurvey extends IHubItemEntity, IWithPermissions {}
```

Comment:
We'll want to define the `hasMapQuestion` and `displayMap` properties on `IHubSurvey` here.

```ts
export interface IHubSurvey extends IHubItemEntity, IWithPermissions {
  hasMapQuestion: boolean;
  displayMap: boolean;
}
```